### PR TITLE
Add aria defaults for lang-en

### PR DESCRIPTION
**Pull request by ClemMakesApps**

### CODE REVIEW

File: src/langEN.js [hunk 1]
```diff
@@ -55,5 +55,37 @@ export default {
             title: 'Save as Image',
             lang: ['Right Click to Save Image']
         }
+    },
+    aria: {
+        general: {
+            withTitle: 'This is a chart about "{title}"',
+            withoutTitle: 'This is a chart，'
```
Comment (Ovilia) [inline src/langEN.js]: Should use `,` instead of `，`. 
Please also check other places.

File: src/langEN.js [hunk 2]
```diff
@@ -55,5 +55,37 @@ export default {
...
+        },
+        series: {
+            single: {
+                prefix: '',
+                withName: 'This is a chart with type {seriesType} named {seriesName}.',
```
Comment (Ovilia) [inline src/langEN.js]: This comes after `general.withTitle` or `general.withoutTitle`, so it should be something like `, and this is ...`.

File: src/langEN.js [hunk 3]
```diff
@@ -55,5 +55,37 @@ export default {
...
+                withoutName: 'This is a chart with type {seriesType}.'
+            },
+            multiple: {
+                prefix: 'This chart consists of {seriesCount} series count.',
+                withName: 'The {seriesId} series is a {seriesType} representing {seriesName}，',
+                withoutName: 'The {seriesId} series is a {seriesType}，',
+                separator: {
+                    middle: '；',
+                    end: '。'
```
Comment (Ovilia) [inline src/langEN.js]: middle: ';`, end: '.'

**Comment by ClemMakesApps** [comment]
I've made the changes based on your feedback. @Ovilia can you review again?

**Comment by Ovilia** [comment]
Thanks @ClemMakesApps . I did a few changes and merged.

**Pull request merged**